.gitignore

```diff
@@ -154,3 +154,4 @@ FirebaseAppCheck/Apps/AppCheckCustomProvideApp/AppCheckCustomProvideApp/GoogleSe
 /Example/FirestoreSample/FirestoreSample/GoogleService-Info.plist
 /Example/FirestoreSample/ui-debug.log
 /Example/FirestoreSample/firestore-debug.log
+FirebaseMessaging/Apps/SampleStandaloneWatchApp/SampleStandaloneWatchApp Watch App/GoogleService-Info.plist
```

Comment:
Can we just use this one?

https://github.com/firebase/firebase-ios-sdk/blob/master/.gitignore#L27

Reply:
The path for this sample is different so had to add this entry. However, I've moved this line within the file so its in same place as (colocated with)  other Messaging gitignores.